Dockerfile

```diff
@@ -1,15 +1,8 @@
-FROM mjstealey/hs_docker_base:1.7.5
+FROM mjstealey/hs_docker_base:1.9.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
-RUN pip install --upgrade pip && pip install \
-  xmltodict==0.10.2 \
-  selenium \
-  dominate \
-  django-robots
-RUN curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-RUN apt-get update && apt-get install -y nodejs
-RUN npm install -g phantomjs-prebuilt
```

Comment:
@mjstealey I remembered we had some issues before when nodejs and phantomjs installation were moved into docker base image, as a result, their installation were put into Dockerfile of the hydroshare container. If those issues have been resolved, these changes look good and a +1 from me, but want to bring this up just in case.